src/groupeddataframe/grouping.jl

```diff
@@ -355,10 +355,19 @@ end
 function aggregate(d::AbstractDataFrame,
                    cols::Union{S, AbstractVector{S}},
                    fs::Union{T, Vector{T}};
-                   sort::Bool=false) where {S<:ColumnIndex, T <:Function}
+                   sort::Bool=false) where {S<:Symbol, T <:Function}
     aggregate(groupby(d, cols, sort=sort), fs)
 end
 
+function aggregate(d::AbstractDataFrame,
+                   cols::Union{S, AbstractVector{S}},
+                   fs::Union{T, Vector{T}};
+                   sort::Bool=false) where {S<:Union{BigInt, Signed, Unsigned}, T <:Function}
+    # Union{BigInt, Signed, Unsigned} needed to exclude Bool
```

Comment:
Why do we need to exclude `Bool`?

Anyway I'd find it better to throw an error for `Bool` inside the function, so that we can use `Integer`. A custom type might be `Integer` but not `Signed` nor `Unsigned`.

Reply:
In the end I have to fix it elsewhere as `aggregate` can take `GroupedDataFrame` created elsewhere. But yes - I was thinking of Julia 0.6, but since 0.7 onwards Base is more careful about indexing with `Bool` so we have what we want for free.